flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
this is disappointing...

Reply:
I can't promise anything, but I'll try to cross-compile locally and see how far I can get. I agree that it would be nice to have this working on aarch64, considering the RPi CM4 exposes PCIe.

Reply:
why don't you let the builders handle it?

Reply:
I can't stop the build job if make fails, and the CI keeps the job alive for ~2 hours.  
I had to fix 3 modules locally already. That's 6 CI hours.